backend/controllers/doctor-controller.js

```diff
@@ -341,6 +286,56 @@ async function getAppointmentsAndPatients(req, res) {
         .catch(err => console.log(err));
 }
 
+async function makeAppointment(req, res) {
+    await db
+        .query(
+            `INSERT INTO Appointment(Patient_PatientId, Doctor_DoctorId, Date, Time, Status)
+                        SELECT ${req.body.patientId} , DoctorId, '${req.body.date}' , '${req.body.time}', 'pending'
+                            FROM Doctor
+                            WHERE User_AccountId=${req.params.userId}`,
```

Comment:
just a question, is there a reason we're passing the user id through the req.params instead of the req.body like all the other ones ?

Reply:
No reason lol is there a way that's better than the other?